src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -643,6 +663,85 @@ verifyMethodSpec cc methodSpec lemmas checkSat tactic asp =
             , post_override_state
             )
 
+
+
+
+refineMethodSpec ::
+  ( ?lc :: Crucible.TypeContext
+  , ?memOpts::Crucible.MemOptions
+  , ?w4EvalTactic :: W4EvalTactic
+  , ?checkAllocSymInit :: Bool
+  , ?singleOverrideSpecialCase :: Bool
+  , Crucible.HasPtrWidth (Crucible.ArchWidth arch)
+  , Crucible.HasLLVMAnn Sym
+  ) =>
+  LLVMCrucibleContext arch ->
+  MS.CrucibleMethodSpecIR (LLVM arch) ->
+  [MS.ProvedSpec (LLVM arch)] ->
+  ProofScript () ->
+  TopLevel (SolverStats, Set TheoremNonce)
+refineMethodSpec cc methodSpec lemmas tactic =
+  ccWithBackend cc $ \bak ->
+  do printOutLnTop Info $
+       unwords ["Verifying", (methodSpec ^. csName) , "..."]
```

Comment:
We could change this message to something more descriptive, so that refinement can be distinguished from "normal" verification in logs.